superset/db_engine_specs.py

```diff
@@ -814,6 +834,164 @@ def get_view_names(cls, inspector, schema):
         """
         return []
 
+    @classmethod
+    def _create_column_info(cls, column, name, data_type):
```

Comment:
Would you mind providing type hints for all arguments and the return type? It's a [suggestion](https://github.com/apache/incubator-superset/blob/master/CONTRIBUTING.md#typing) we have for all new Python methods and helps with type-checking and readability.

Reply:
Definitely!

Reply:
I added types but then I got errors for other functions in the file. It looks like once typing is added, then the entire file needs to be typed? Since it is a huge file, I'd like to have that refactoring done in a separate PR.

Reply:
I was able to add type hints and returns for all new methods except for the my_db parameter in the method select_star because it exposed a circular dependency. I created issue https://github.com/apache/incubator-superset/issues/7412 to address refactoring for this file.